core/config.go

```diff
@@ -141,6 +142,7 @@ func setFlags() error {
 	pflag.String("metrics.address", viper.GetString("metrics.address"), "IP of metrics server")
 	pflag.Int("metrics.port", viper.GetInt("metrics.port"), "Port of metrics server (listens on server.address)")
 	pflag.String("metrics.path", viper.GetString("metrics.path"), "URI path to metrics endpoint")
+	pflag.String("reposSshKey", viper.GetString("reposSshKey"), "Path to a readable private ssh key for repo operations")
```

Comment:
Should the private key end up in `/etc/o2.d/aliecs`, as it doesn't need to be writable